sdk/lib/drivers/sound/mmebuddy/wave/wodMessage.c

```diff
@@ -118,6 +118,22 @@ wodMessage(
             break;
         }
 
+        case WODM_GETVOLUME :
+        {
+            Result = MmeGetVolume(WAVE_OUT_DEVICE_TYPE,
+                                  DeviceId,
+                                  PrivateHandle,
+                                  &Parameter1);
+        }
+
+        case WODM_SETVOLUME :
```

Comment:
Is it really intended to not have a break between `WODM_GETVOLUME` and `case WODM_SETVOLUME`?
If so, I would add a comment, something like `// fall-through intended`.
Otherwise ofc better to add the `break;` ;-)

Reply:
Ah, indeed! I forgot to add a break. Thank you! :sweat_smile: